cinema/views.py

```diff
@@ -33,7 +33,7 @@ def get_serializer_class(self):
 
 
 class MovieSessionViewSet(viewsets.ModelViewSet):
-    queryset = MovieSession.objects.all().select_related(
+    queryset = MovieSession.objects.select_related(
```

Comment:
and better use one select_related -  MovieSession.objects.select_related("movie", "cinema_hall")